setup.py

```diff
@@ -38,46 +38,24 @@ def is_platform_mac():
 except ImportError:
     _CYTHON_INSTALLED = False
 
-try:
-    import pkg_resources
-    from setuptools import setup, Command
-    _have_setuptools = True
-except ImportError:
-    # no setuptools installed
-    from distutils.core import setup, Command
-    _have_setuptools = False
 
-setuptools_kwargs = {}
+from setuptools import setup, Command
+import pkg_resources
+
+
 min_numpy_ver = '1.9.0'
-if sys.version_info[0] >= 3:
 
-    setuptools_kwargs = {'zip_safe': False,
-                         'install_requires': ['python-dateutil >= 2',
-                                              'pytz >= 2011k',
-                                              'numpy >= %s' % min_numpy_ver],
-                         'setup_requires': ['numpy >= %s' % min_numpy_ver]}
-    if not _have_setuptools:
-        sys.exit("need setuptools/distribute for Py3k"
-                 "\n$ pip install distribute")
+setuptools_kwargs = {'test_suite': 'nose.collector',
```

Comment:
why did you add ``nose.collector``?

Reply:
I would completely remove this

Reply:
> why did you add nose.collector?

It was already there, just lower in the file.

Reply:
I know, but I don't think its needed. remove and let's see.

Reply:
we *should* have a test_requires, but that's separate. can you open an issue for that.